webviewer/src/pages/Account/components/Profile/Profile.jsx

```diff
@@ -1,36 +1,66 @@
 import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import { selectUser, useUserActions } from 'state';
+import { selectUser, useUserActions, useInterfaceActions } from 'state';
 import { ROUTES } from '_constants';
-import { InfoItem, InfoTitle, InfoValue, Title, StyledButton } from './Profile.styles';
 import { ProfileModal } from './ProfileModal';
+import { Button } from 'components/_shared/Button';
+import {
+  InfoItem,
+  InfoTitle,
+  InfoValue,
+  Title,
+  StyledButton,
+  UserMessage
+} from './Profile.styles';
 
 export const Profile = () => {
   const user = useSelector(selectUser);
   const history = useHistory();
-  const { toggleLogoutModal } = useUserActions();
+  const { toggleLogoutModal, updateUser } = useUserActions();
+  const { setOpenContactUs } = useInterfaceActions();
   const [isOpenModal, setIsOpenModal] = useState(false);
 
   const toggleModal = () => setIsOpenModal(!isOpenModal);
   const handleChangePassword = () => {
     history.push(ROUTES.RESET_PASSWORD);
   };
+  const handleTopUp = () => {
+    setOpenContactUs(true);
+    if (user.trial_finished_at && !user.is_trial_end_notified)
+      updateUser({ is_trial_end_notified: true });
```

Comment:
such an advice. use the separate file for API properties, like a DataProperties or something like this.cause it will make your code more cleaner and readable.